src/renderer/components/item-object-list/item-list-layout.tsx

```diff
@@ -197,6 +212,20 @@ export class ItemListLayout extends React.Component<ItemListLayoutProps> {
     return this.applyFilters(filterItems, allItems);
   }
 
+  updateColumnFilter(checkboxValue: boolean, index: number) {
+    if (checkboxValue){
+      this.columnFilter = this.columnFilter | ( 1 << index );
+    } else {
+      this.columnFilter = this.columnFilter ^ ( 1 << index );
+    }
+  }
```

Comment:
Why all this complications about numbers? Since the goal of the feature is to only show specific columns. Wouldn't just a `Set<ColumnId>` be sufficiant (and then stored in the store as `ColumnId[]`)?

Reply:
@Nokel81 Thanks for your comments, how do you feel about storing all columns settings as Record<String, Object> on user-store.ts side and work with the object here as map<string, props_obj> where map key is column name and props is a container of all related properties (visibility for now, wight, order, etc. for the future)?
I think using column names instead of ids is safer and more user-friendly

Reply:
I definitely agree that column names are better than ids. I wonder why the store shouldn't just be `string[]` and then in the code it be a `Set<string>`? This way it seems simpler than doing an object with a `visible` field.

Reply:
Hmm, I was sure that we should show New columns, which will be added in master code but not hidden by the user manually

Reply:
@Nokel81 Fixed. One more question, how about ignoring table config saving on 
`this.columnProps = = this.props.renderTableHeader.map(cellProps => { return cellProps.className; })`
I mean the case when the user didn't change anything, I think we can skip columns saving and load those from renderTableHeader on the next component mounting

Also, I would like to do something with unlabeled columns (like <warning>), my idea is about linking these columns behavior with the nearest labeled columns (for instance pods.name + pods.warning). Does it make sense?

Reply:
> Hmm, I was sure that we should show New columns, which will be added in master code but not hidden by the user manually

I completely agree, I guess I would then store "hidden" columns.

Reply:
Ok, will fix it today